src/libs/Launcher.js

```diff
@@ -169,17 +189,71 @@ export default class Launcher {
         .updateAttributes(folderId, { name: sourceAccountIdentifier })
     } catch (err) {
       log.warn(
-        `Could not rename the destination folder ${folderId} to ${sourceAccountIdentifier}. ${err.message}`
+        `Could not rename the destination folder ${folderId} to ${sourceAccountIdentifier}. ${err}`
       )
     }
   }
 
+  /**
+   * Ensures that account and triggers are created and launch the trigger
+   */
+  async ensureAccountTriggerAndLaunch() {
+    const startContext = this.getStartContext()
+    let { trigger, account, connector, client, job, launcherClient } =
+      startContext
+
+    if (!account) {
+      log.debug(
+        `ensureAccountAndTriggerAndJob: found no account in start context. Creating one`
+      )
+      const accountResponse = await client.create(
+        'io.cozy.accounts',
+        models.account.buildAccount(connector, {})
+      )
+      account = accountResponse.data
+      log.debug(`ensureAccountAndTriggerAndJob: created account`, account)
+    }
+    if (!trigger) {
+      log.debug(
+        `ensureAccountAndTriggerAndJob: found no trigger in start context. Creating one`
+      )
+      const triggerResponse = await client.create(
+        'io.cozy.triggers',
+        models.trigger.triggers.buildTriggerAttributes({
+          account,
+          konnector: connector
+        })
```

Comment:
same here. create() is only useful if you use the third arg (aka reference), if not, save is the way to go